Module not found: Can't resolve 'css-select'

When attempting to use the package (htmlToSlate()), I am getting execution error. 

```
Module not found: Can't resolve 'css-select'
error - ./node_modules/slate-serializers/lib/serializers/htmlToSlate/index.js:8:21

```

Installing the _css-select_ package manually seems to solve the problem but might be worth to add it as a required dependency.

Thank you for your time working on this package btw.
Appreciate it!

--- a/package.json
+++ b/package.json
@@ -32,7 +32,6 @@
     "@types/html-escaper": "^3.0.0",
     "@types/jest": "^29.4.0",
     "commitizen": "^4.3.0",
-    "css-select": "^5.1.0",
     "cz-conventional-changelog": "^3.3.0",
     "husky": "^8.0.3",
     "jest": "^29.4.0",
@@ -54,6 +53,7 @@
     ]
   },
   "dependencies": {
+    "css-select": "^5.1.0",
     "dom-serializer": "^2.0.0",
     "domhandler": "^5.0.3",
     "domutils": "^3.0.1",

